nixos/tests/containers-extra_veth.nix

```diff
@@ -0,0 +1,103 @@
+# Test for NixOS' container support.
```

Comment:
I might be missing something, but I can't see where this is added to the release tests?